tests/integration/test_dbstatecheck.py

```diff
@@ -79,27 +79,27 @@ def test_basic(checker):
         ['output', '10010101T0000Z', 'succeeded'],
         ['good', '10000101T0000Z', 'waiting', '(flows=2)'],
         ['good', '10010101T0000Z', 'waiting', '(flows=2)'], ]
-    assert result == expect
+    assert sorted(result) == sorted(expect)
```

Comment:
What does this have to do with this PR?

Reply:
I think that this was flaky. Happy to revert if you think it unreasonable.